distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+
+.Example: Create a Directory and Switch User on *NIX
+----
+mkdir new_installation
+chown ${branding-lowercase}-user:${branding-lowercase}-group new_installation
+
+su - ${branding-lowercase}-user
+----
+
+* Change the current directory to location of zip file (${branding-lowercase}-X.Y.zip).
+
+.Example: Where the Zip File may be Located in *NIX
+----
+cd /home/user/cdrom
+----
+
+.Windows (Example assumes ${branding} has been downloaded to the D drive)
+----
+cd D:\
+----
+* Copy ${branding-lowercase}-X.Y.zip to <INSTALL_DIRECTORY>.
+
+.*NIX
+----
+cp ${branding-lowercase}-X.Y.zip <INSTALL_DIRECTORY>
+----
+
+.Windows
+----
+copy ${branding-lowercase}-X.Y.zip <INSTALL_DIRECTORY>
+----
+
+* Change the current directory to the desired install location.
+
+.*NIX or Windows
+----
+cd <INSTALL_DIRECTORY>
+----
+
+* The ${branding} zip is now located within the `<INSTALL_DIRECTORY>`. Unzip ${branding-lowercase}-X.Y.zip.
+
+.*NIX
+----
+unzip ${branding-lowercase}-X.Y.zip
+----
+
+.Example: Use Java to Unzip in Windows
+----
+"C:\Program Files\Java\jdk1.8.0\bin\jar.exe" xf ${branding-lowercase}-X.Y.zip
+----
+
+* Run ${branding} using the appropriate script.
+
+.*NIX
+----
+<INSTALL_DIRECTORY>/${branding-lowercase}-X.Y/bin/${branding-lowercase}
+----
+
+.Windows
+----
+<INSTALL_DIRECTORY>/${branding-lowercase}-X.Y/bin/${branding-lowercase}.bat
+----
+
+Execute the following command at the command line for status:
+
+.View Status
+----
+${branding-lowercase}${at-symbol}local>list
+----
+
+// Verify this step is still required.
+* If the ${branding} Standalone Solr Server will be installed later, an additional configuration step is required for the ${branding} kernel.
+Add the following lines to the bottom of the `<INSTALL_DIR>/etc/org.ops4j.pax.web.cfg` file:
+
+.Additional Configuration Step
+----
+# Jetty Configuration
+`org.ops4j.pax.web.config.file=<KARAF.HOME>/etc/jetty.xml`
+----
+
+* Run the ${branding} using the appropriate script.
+
+.*NIX
+----
+<INSTALL_DIRECTORY>/${branding-lowercase}-${project.version}/bin/${branding-lowercase}
+----
+
+.Windows
+----
+<INSTALL_DIRECTORY>/${branding-lowercase}-${project.version}/bin/${branding-lowercase}.bat
+----
+
+The distribution takes a few moments to load depending on the hardware configuration.
+Execute the following command at the command line for status:
+
+.View Status
+----
+${branding}${at-symbol}local>list
+----
+The list of bundles should look similar to this:
+
+.${branding} List of Apps Installed
+----
+.${branding}${at-symbol}local>list
+START LEVEL 100 , List Threshold: 50
+   ID   State         Blueprint      Spring    Level  Name
+[ 111] [Active     ] [            ] [       ] [   80] Commons IO (2.1.0)
+[ 113] [Active     ] [Created     ] [       ] [   80] ${branding} :: Distribution :: Console Branding Plugin (${project.version})
+----
+
+.${branding} Application Installation Dependencies
```

Comment:
This step is a little risky. If the user runs the `app:tree` too soon, DDF won't be done loading and the command will fail or will report just some of the dependencies. I'm also not sure how useful this is at this point in the setup process. Maybe we should consider moving after the installer has been run?